XMonad/Actions/DynamicWorkspaceGroups.hs

```diff
@@ -145,3 +158,20 @@ promptWSGroupForget :: XPConfig -> String -> X ()
 promptWSGroupForget xp s = do
   gs <- fmap (M.keys . unWSG) XS.get
   mkXPrompt (WSGPrompt s) xp (mkComplFunFromList' xp gs) forgetWSGroup
+
+-- $topics
+-- You can use this module with "XMonad.Actions.TopicSpace" - just replace
```

Comment:
If you want to be correct to an obnoxious degree, this should be an em dash (—).  This is usually emulated in ASCII as three dashes `---`.  If not just ignore :)